program/src/admin_set_parameters.rs

```diff
@@ -0,0 +1,67 @@
+use jito_bytemuck::{types::PodU64, AccountDeserialize, Discriminator};
+use jito_jsm_core::loader::load_signer;
+use jito_restaking_core::{config::Config, ncn::Ncn};
+use jito_tip_router_core::{
+    constants::{
+        MAX_EPOCHS_BEFORE_STALL, MAX_SLOTS_AFTER_CONSENSUS, MIN_EPOCHS_BEFORE_STALL,
+        MIN_SLOTS_AFTER_CONSENSUS,
+    },
+    error::TipRouterError,
+    ncn_config::NcnConfig,
+};
+use solana_program::{
+    account_info::AccountInfo, entrypoint::ProgramResult, msg, program_error::ProgramError,
+    pubkey::Pubkey,
+};
+
+pub fn process_admin_set_parameters(
+    program_id: &Pubkey,
+    accounts: &[AccountInfo],
+    epochs_before_stall: Option<u64>,
+    valid_slots_after_consensus: Option<u64>,
+) -> ProgramResult {
+    let [restaking_config, config, ncn_account, ncn_admin, restaking_program] = accounts else {
+        return Err(ProgramError::NotEnoughAccountKeys);
+    };
+
+    load_signer(ncn_admin, true)?;
+
+    // Load and verify accounts
+    NcnConfig::load(program_id, ncn_account.key, config, true)?;
+    Ncn::load(restaking_program.key, ncn_account, false)?;
+    Config::load(restaking_program.key, restaking_config, false)?;
+
+    let ncn_data = ncn_account.data.borrow();
```

Comment:
Super dumb nit - ignore if you want to. I like to surround blocks like this with brackets and add a comment